SimExLite/SourceCalculators/GaussianSourceCalculator.py

```diff
@@ -0,0 +1,207 @@
+""":module GaussianSourceCalculator: Module that holds the GaussianSourceCalculator class.  """
+
+import sys
+import numpy as np
+from pathlib import Path
+from scipy.constants import hbar, c
+from pint import Quantity, Unit
+import imp
+
```

Comment:
General: imports should be sorted alphabetically, `from .. import ..` go after plain `import ..`s. Check if imported modules are unused.

Reply:
Besides to sort them alphabetically, I would like to have the global package first and the local package after.